Tests/PointFreeTests/EmailTests/__Snapshots__/NewEpisodeEmailTests/testNewEpisodeEmail_Announcement_NonSubscriber.2.txt

```diff
@@ -11,9 +11,9 @@ text, no markdown allowed. Here is some more text just to have some filler.
 
  <http://localhost:8080/episodes/ep1-proof-in-functions>
 
-This episode is for subscribers only. To access it, and all past and future episodes, become a subscriber today!
+This episode is free for everyone, made possible by our subscribers. Consider becoming a subscriber today!
```

Comment:
this doesn't seem right. did more mock data change? we might want to revert some of the mock data and write separate tests for it?

Reply:
Yeah. I added to the main mock to have some variety but need to clean things up. May just localize the `.free` episode to the new tests and address test diversity later.